README.md

```diff
@@ -177,6 +177,7 @@ A categorized community-driven collection of awesome Pharo libraries, tools, fra
 ## Web
 + [ChartJs](https://github.com/DuneSt/ChartJs) - Seaside binding of ChartJs to display and interact with charts.
 + [MaterialDesignLite](https://github.com/DuneSt/MaterialDesignLite) - Binding google's Material Design Lite project for Seaside.
++ [OpenAPI](https://github.com/zweidenker/OpenAPI) - A pharo implementation of OpenAPI 3.0.1.
```

Comment:
Maybe we could add a short description of what is OpenAPI?

Reply:
```suggestion
+ [OpenAPI](https://github.com/zweidenker/OpenAPI) - A pharo implementation of [OpenAPI](https://www.openapis.org) 3.0.1, a specification for machine-readable interface files for describing, producing, consuming, and visualizing RESTful web services.
```

Reply:
@jecisc  bump.